.github/workflows/deploy.yaml

```diff
@@ -107,22 +96,12 @@ jobs:
 
             *Pusher: @${{ github.actor }}, Action: \`${{ github.event_name }}\`, Working Directory: \`${{ env.tf_actions_working_dir }}\`, Workflow: \`${{ github.workflow }}\`*`;
 
-            // 3. If we have a comment, update it, otherwise create a new one
-            if (botComment) {
-              github.rest.issues.updateComment({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                comment_id: botComment.id,
-                body: output
-              })
-            } else {
-              github.rest.issues.createComment({
-                issue_number: context.issue.number,
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                body: output
-              })
-            }
+            github.rest.issues.createComment({
```

Comment:
So moving forward it will always create a new comment instead of updating an existing one?

Reply:
Ah looking at this PR it would appear so - I think I prefer the previous approach tbh as it adds less noise.

Reply:
> Ah looking at this PR it would appear so - I think I prefer the previous approach tbh as it adds less noise.

I thought this too... but then I became confused as to what the "current state" was. So I switched back. It does create a lot of noise like you said. I'm fine with switching it back

Reply:
Resolved in latest commit. Reverted back to previous approach.